### PR TITLE
Fran0571 _table feature 

### PR DESCRIPTION
**Pull request by fran0571**
This includes:
1. Creation of a new branch.
2. Addition of HTML code and styling for the tables feature

### CODE REVIEW

File: src/_tables.scss [hunk 1]
```diff
@@ -0,0 +1,34 @@
+//Mixin for table styles
+@use 'sass:color';
+
+@mixin table-styles {
```
Comment (le000255) [inline src/_tables.scss]: Same here with this mixin. Please move it into `_mixins.scss`, then push your changes. Thank you!

**Review by le000255 — CHANGES_REQUESTED** [review]
I left a comment for this PR, please read below.

File: src/_tables.scss (hunk 1 above)
Comment (fran0571) [reply src/_tables.scss]: Perfect, changes done and pushed.

**Review by le000255 — APPROVED** [review]

**Comment by le000255** [comment]
Conflicts resolved!

**Pull request merged**